.gitlab-ci.yml

```diff
@@ -100,32 +100,32 @@ default:
         ENV: ${SERVICE_VAULT_ENV}
         TEST: ITVaultTest
   rules:
-    - if: '$CLOUD == "GCP" && $TEST == "ITFileScanTest"'
+    - if: $CI_COMMIT_BRANCH && '$CLOUD == "GCP" && $TEST == "ITFileScanTest"'
       allow_failure: true
-    - if: '$CLOUD == "GCP" && $TEST != "ITFileScanTest"'
+    - if: $CI_COMMIT_BRANCH && '$CLOUD == "GCP" && $TEST != "ITFileScanTest"'
       allow_failure: false
-    - if: '$CLOUD != "GCP"'
+    - if: $CI_COMMIT_BRANCH && '$CLOUD != "GCP"'
       allow_failure: false
 
-dotnet-sdk-lint:
-  extends: .dotnet-sdk-base
+sdk-lint:
+  extends: .sdk-base
   stage: lint
   script:
     - dotnet format --verify-no-changes
 
-dotnet-sdk-unit-tests:
-  extends: .dotnet-sdk-base
+sdk-unit-tests:
+  extends: .sdk-base
   stage: unit_tests
   script:
     - dotnet test --no-restore --filter SignerTest
 
-dotnet-sdk-integration-tests:
-  extends: .dotnet-sdk-test-base
+sdk-integration-tests:
+  extends: .sdk-test-base
   script:
     - dotnet test --no-restore --filter ${TEST}
 
-.dotnet-sdk-staging-tests:
-  extends: .dotnet-sdk-test-base
+.sdk-staging-tests:
```

Comment:
@pangea-andrest Is this supposed to be a running job? Because at the moment it begins with `.` which makes it a config section instead, and nothing is `extends`-ing it.

Reply:
No, it isn't. It was thought in order to be imported in staging tests (pre roll out to prod), but we should remove it due to it's not used

Reply:
Will do in a future PR.

Reply:
https://github.com/pangeacyber/pangea-dotnet/pull/215